web_responsive/static/src/xml/form_view.xml

```diff
@@ -138,4 +138,14 @@
         </t>
     </t>
 
+    <t t-extend="mail.Chatter">
+        <t t-jquery=".o_chatter_topbar" t-operation="replace">
+            <div class="o_chatter_header_container">
+                <div class="o_chatter_topbar">
```

Comment:
Use `<t t-raw="0"/>` instead. It's the system they added to wrap elements.

Reply:
How? i don't know how use raw="0" in this context :S

Reply:
It allows to move the target element that is being replaced to a different place. In your case, it would be:

```xml
<t t-jquery=".o_chatter_topbar" t-operation="replace">
    <div class="o_chatter_header_container">
        <t t-raw="0"/>
    </div>
</t>
```

Reply:
Ummm doesn't work... I understand the "t-raw='0'" on a "t-call" context... but here i can't see it :/

This change, without something like "t-raw='0'", is incompatible with other modules that modify the chatter, right? It will depend on the order of the dependencies/installations...

Reply:
Yikes! Sorry I was mixing 2 concepts. :confounded: 

You're right. I was meaning to use [`position="move"`](https://www.odoo.com/documentation/12.0/reference/views.html#inheritance-specs), but after a little research, it seems that's only available for backend Qweb, so there's nothing that can be improved here.